ccd-definition/CaseEventToComplexTypes/draft-CMO/caseManagementOrder.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CaseManagementOrder",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "caseManagementOrder",
+    "ListElementCode": "hearingDate",
```

Comment:
why does this need to be defined in caseEventToComplexTypes?

Reply:
Needs to display caseManagementOrder updates in draft order tab, which is the reason I created this complex field and associated with caseEvent. followed the same pattern as SDO.